code/_onclick/hud/fullscreen.dm

```diff
@@ -92,6 +92,13 @@
 		fs_view = client_view
 		transform = matrix(actualview[1]/FULLSCREEN_OVERLAY_RESOLUTION_X, 0, 0, 0, actualview[2]/FULLSCREEN_OVERLAY_RESOLUTION_Y, 0)
 
+/obj/screen/fullscreen/black
+	icon_state = "black" //just a black square, you can change this if you get better ideas
+	layer = FULLSCREEN_INTRO_LAYER
+
+/obj/screen/fullscreen/spawn
+	icon_state = "blackimageoverlay" //just a black square, you can change this if you get better ideas
```

Comment:
Are these really both squares?

If yes why arent they the same typepath?

Reply:
whoops, my bad. fixing it.